Not passed test

Not passed test `cli when supplied with report output file path should write the file and create dirs if they don't exist:`

**Tell us about your environment**

* **ESLint Version: 3.12.2
* **Node Version: 7.2.0
* **npm Version: 3.10.9

**What parser (default, Babel-ESLint, etc.) are you using? Default

**What did you do? Please include the actual source code causing the issue.**

```bash
$ npm test
```

**What did you expect to happen?**

all passed tests

**What actually happened? Please include the actual, raw output from ESLint.**

```sh
  13566 passing (1m)
  1 failing

  1) cli when supplied with report output file path should write the file and create dirs if they don't exist:
     AssertionError: expected '\nC:\\Users\\ivdemidov\\AppData\\Local\\Temp\\eslint\\fixtures\\single-quoted.js\n  1:13  warning  Strings must use doublequote  quotes\n\n✖ 1 problem (0 errors, 1 warning)\n' to include 'C:\\Users\\IVDEMI~1\\AppData\\Local\\Temp\\eslint\\fixtures\\single-quoted.js'
      at Function.assert.include (D:\sand-box\git\eslint\node_modules\chai\lib\chai\interface\assert.js:843:45)
      at Context.it (D:\sand-box\git\eslint\tests\lib\cli.js:541:20)
      at callFn (D:\sand-box\git\eslint\node_modules\mocha\lib\runnable.js:326:21)
      at Test.Runnable.run (D:\sand-box\git\eslint\node_modules\mocha\lib\runnable.js:319:7)
      at Runner.runTest (D:\sand-box\git\eslint\node_modules\mocha\lib\runner.js:422:10)
      at D:\sand-box\git\eslint\node_modules\mocha\lib\runner.js:528:12
      at next (D:\sand-box\git\eslint\node_modules\mocha\lib\runner.js:342:14)
      at D:\sand-box\git\eslint\node_modules\mocha\lib\runner.js:352:7
      at next (D:\sand-box\git\eslint\node_modules\mocha\lib\runner.js:284:14)
      at Immediate.<anonymous> (D:\sand-box\git\eslint\node_modules\mocha\lib\runner.js:320:5)



=============================================================================
Writing coverage object [D:\sand-box\git\eslint\coverage\coverage.json]
Writing coverage reports at [D:\sand-box\git\eslint\coverage]
=============================================================================

=============================== Coverage summary ===============================
Statements   : 99.48% ( 11563/11624 ), 147 ignored
Branches     : 98.2% ( 8891/9054 ), 75 ignored
Functions    : 99.9% ( 1999/2001 ), 22 ignored
Lines        : 99.47% ( 11490/11551 )
================================================================================

START:
20 12 2016 16:17:43.628:WARN [karma]: Port 9876 in use
20 12 2016 16:17:43.631:INFO [karma]: Karma v0.13.22 server started at http://localhost:9877/
20 12 2016 16:17:43.636:INFO [launcher]: Starting browser PhantomJS
20 12 2016 16:17:45.261:INFO [PhantomJS 2.1.1 (Windows 8 0.0.0)]: Connected on socket ep7LPmVfCAehOM8-AAAA with id 88475412

Finished in 0.644 secs / 0.589 secs

SUMMARY:
√ 233 tests completed
npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -98,7 +98,7 @@
     "load-perf": "^0.2.0",
     "markdownlint": "^0.3.1",
     "mocha": "^2.4.5",
-    "mock-fs": "^3.12.1",
+    "mock-fs": "^4.0.0",
     "npm-license": "^0.3.2",
     "phantomjs-prebuilt": "^2.1.7",
     "proxyquire": "^1.7.10",

